cirq/google/engine/engine.py

```diff
@@ -518,6 +519,54 @@ def list_programs(self,
             ) for p in response
         ]
 
+    def list_jobs(self,
+                  created_before: Optional[
+                      Union[datetime.datetime, datetime.date]] = None,
+                  created_after: Optional[
+                      Union[datetime.datetime, datetime.date]] = None,
+                  has_labels: Optional[Dict[str, str]] = None,
+                  execution_states: Optional[Set[
+                      quantum.enums.ExecutionStatus.State]] = None):
+        """Returns the list of jobs in the project.
+
+        All historical jobs can be retrieved using this method and filtering
+        options are available too, to narrow down the search baesd on:
+          * creation time
+          * job labels
+          * execution states
+
+        Args:
+            created_after: retrieve jobs that were created after this date
+                or time.
+            created_before: retrieve jobs that were created after this date
+                or time.
+            has_labels: retrieve jobs that have labels on them specified by
+                this dict. If the value is set to `*`, filters having the label
+                regardless of the label value will be filtered. For example, to
+                query programs that have the shape label and have the color
+                label with value red can be queried using
+                `{'color: red', 'shape:*'}`
```

Comment:
whitespace is inconsistent here.  Is this intended?  Also, this is a set, not a dict.  Is it supposed to be:

{'color': 'red', 'shape': '*'}

Reply:
Oh yeah, it totally should be a dict

Reply:
> whitespace is inconsistent here. Is this intended?

What do you mean by that? The newline before the dict? It would go over 80 chars otherwise, hence the newline there. Other methods use 4 spaces for the indentation of multiline Args docstring, so that is consistent.

Reply:
I meant that it was 'color:**_**red' but 'shape:*'.  It doesn't matter since it should be a dict.

Reply:
Got it!!! 👍